src/Assignments/ProblemDetails.tsx

```diff
@@ -13,12 +13,14 @@ interface ProblemDetailsProps {
     problem: ProblemObject;
     topic: TopicObject | null;
     attemptsRemaining?: number;
+    setAttemptsRemaining?: (a: number)=>unknown;
```

Comment:
If this is a setState from useState, the type can be a bit more explicit, but that's not dreadfully important.